app/models/qcable_creator.rb

```diff
@@ -9,22 +9,26 @@ class QcableCreator < ApplicationRecord # rubocop:todo Style/Documentation
   validates :user, presence: true
   validates :lot, presence: true
 
-  attr_accessor :count, :barcodes
+  attr_accessor :count, :barcodes, :supplied_barcode
 
   after_create :make_qcables!
 
   def make_qcables!
     qcables_by_count! if count.present?
     qcables_by_barcode! if barcodes.present?
+    qcables_by_supplied_barcode! if supplied_barcode.present?
```

Comment:
should we add this in the first line as:
```suggestion
    return qcables_by_supplied_barcode! if supplied_barcode.present?
```
I'm thinking we don't want to merge all the behaviour, but to add an optional branch for it

Reply:
Not sure about the return but what I mean is to not run the other lines when supplied_barcode is present